source/extensions/transport_sockets/tls/cert_validator/utility.cc

```diff
@@ -5,7 +5,12 @@ namespace Extensions {
 namespace TransportSockets {
 namespace Tls {
 
-int CertValidatorUtil::ignoreCertificateExpirationCallback(int ok, X509_STORE_CTX* store_ctx) {
+// When the minimum supported BoringSSL includes
+// https://boringssl-review.googlesource.com/c/boringssl/+/53965, remove this and have
+// callers just set `X509_V_FLAG_NO_CHECK_TIME` directly.
```

Comment:
Can you add a compile-time failure for when that happens, so we don't forget to do this? Something like 

```
#if defined(BORINGSSL_FIPS) && defined(X509_V_FLAG_NO_CHECK_TIME)
#error "Remove legacy codepath"
#endif
```

Reply:
Sure, a comment in repository_locations sounds good.

Reply:
Done